patches/minecraft/net/minecraft/server/management/PlayerList.java.patch

```diff
@@ -146,14 +146,20 @@
          return entityplayermp;
      }
  
-@@ -556,15 +601,20 @@
+@@ -556,15 +601,26 @@
  
      public void func_187242_a(EntityPlayerMP p_187242_1_, int p_187242_2_)
      {
 +        transferPlayerToDimension(p_187242_1_, p_187242_2_, field_72400_f.func_71218_a(p_187242_2_).func_85176_s());
 +    }
 +
++    // TODO: Remove (1.13)
```

Comment:
Should this not be deprecated instead so people actually notice it before it gets removed?